CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (x/bank) [#24431](https://github.com/cosmos/cosmos-sdk/pull/24431) Reduce the number of `ValidateDenom` calls in `bank.SendCoins` and `Coin`.
```

Comment:
i think we should also mention here that `AmountOf` will no longer panic given an invalid denom.

Reply:
[2978fa0](https://github.com/cosmos/cosmos-sdk/pull/24431/commits/2978fa0b396b7c463b9539fcc84dc5476bd01eba)